File: Frontend/src/app/models/admin/paginator-payload.ts
import { Branch } from "./branch";

export class PaginatorPayLoad {
    user_id?: number;
    totalRecords: number = 0;
    currentPage: number = 0; 
    pageSize: number = 5;
    event_length: number = 0;
    branch_id?: number;
  }

File: Backend/TaxManagmentSystem/src/main/java/com/afr/fms/Common/Entity/PaginatorPayLoad.java
package com.afr.fms.Common.Entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PaginatorPayLoad {
    private Long user_id;
    private int totalRecords;;
    private int currentPage; 
    private int pageSize;
    private String event_length;
    private Long branch_id;
}
